src/components/VitalStat.vue

```diff
@@ -1,28 +1,30 @@
 <script>
 /**
- *   Displays a read-only stat
+ * Displays one stat with @param stat.id as a name of stat
+ * Then @param scale number of points with css style @param stat.style
+ * And init and fill css style depending on computed @param value while looking
+ * at @param stat.defaultValue and @param stat.defaultModifier
  * @param stat {JSON} displayed object
- * @DEPRECATED @param styleProp {String} style of displayed stat points
  * @param scale {Number} total number of points, split into groups of five
  * @param value {Number} sum of dependency values
  */
 export default {
   props: { stat: Object, scale: Number, value: Number },
   data() {
     return {
+      // initalValue is stat.defaultValue or this.finalValue
+      // finalValue is stat.defaultModifier + value or value
       initialValue: this.stat.defaultValue
         ? this.stat.defaultValue
-        : this.finalValue(),
+        : this.finalValue()
     };
   },
   methods: {
     /**
-     * @returns sum of dependecies + modifier
+     * @returns computed modifier + value
```

Comment:
to nieje pravda... modifier su data, tzn ze neni computed
a value je v podstate sum of dependecies (aj ked sa to vola value lebo ten kto to pisal tomu nevymyslel dobry nazov)

Reply:
njn, napisal som to naopak, sorry